### DISCUSSION TITLE
[211820]Starbound {本体}

### DISCUSSION THREAD

**Discussion by YGShaogezhu** [pjy612/SteamManifestCache#609] [Share]
### Steam 游戏ID

211820

### 密钥列表

```
211821;d38d22b9989c3c2c74a3ff2b9164637cd6816f06bbac1cbd57a79098f56e7a49
211822;f34154fb8d8c1bad83ea48eca72eb672004cd9c334cf1042c380b1439b0ec9f2
211823;f557c69f92ae74f4d8658c8633837fd6605f355606ee5381330b63ed15553dc0
```

### 清单附件
[211821_612190494005252909.zip](https://github.com/pjy612/SteamManifestCache/files/15047342/211821_612190494005252909.zip)

### 备注

email:oyqy28404840511@outlook.com
code:54D2FE33412597E0

**Comment by YGShaogezhu** [comment]
第一次尝试分享不知道对不对

**Comment by pjy612** [comment]
* 缺主程序的清单和密钥吧？ https://steamdb.info/app/211820/depots/

**Reply by YGShaogezhu** [reply]
已经更新完成

**Comment by YGShaogezhu** [comment]
已经更新完成
________________________________
发件人: pjy612 ***@***.***>
发送时间: 2024年4月19日 22:46
收件人: pjy612/SteamManifestCache ***@***.***>
抄送: YGShaogezhu ***@***.***>; Author ***@***.***>
主题: Re: [pjy612/SteamManifestCache] [211820]Starbound {本体} (Discussion #609)


  *   缺主程序的清单和密钥吧？ https://steamdb.info/app/211820/depots/

―

**Reply by pjy612** [reply]
* 清单没上传完

**Comment by YGShaogezhu** [comment]
这回应该是没什么问题了
________________________________
发件人: pjy612 ***@***.***>
发送时间: 2024年4月20日 1:56
收件人: pjy612/SteamManifestCache ***@***.***>
抄送: YGShaogezhu ***@***.***>; Author ***@***.***>
主题: Re: [pjy612/SteamManifestCache] [211820]Starbound {本体} (Discussion #609)


  *   清单没上传完

―
